app/controllers/api/v1/comments_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Api
+  module V1
+    class CommentsController < ApplicationController
+      def index
+        user = User.find(params[:user_id])
+        post = user.posts.find(params[:post_id])
+        comments = post.comments
+        render json: comments
+      end
+
+      def create
+        user = User.find(params[:user_id])
+        post = user.posts.find(params[:post_id])
+        comment = post.comments.new(comment_params)
+        comment.user = current_user
+
+        if comment.save
+          render json: comment, status: :created
+        else
+          render json: { errors: comment.errors }, status: :unprocessable_entity
+        end
+      end
```

Comment:
- [ ] Your create comment action is not correctly created. The route you defined is `POST api/v1/posts/post_id/comments`, but on line 12 you are expecting a `user_id` which does not exist. Kindly embed the comments route within the user posts routes you defined in the routes file if you wish to use the `user_id`.

- [ ] Kindly ensure the comment create action works standalone. You are using `current_user` which API does not really recognize since it is a session saved in the browser by Devise and so Postman does not have access to it. You can use the `user_id` to set the `current_user` instead of using Devise's.

  ![image](https://github.com/mershark/blog-app/assets/54848363/a9bf0df5-b897-4efb-841a-d7e0d10a809b)